packages/core/src/agent/Agent.ts

```diff
@@ -98,7 +98,7 @@ export class Agent extends BaseAgent {
     // set the pools on the ledger.
     this.ledger.setPools(this.ledger.config.indyLedgers)
     // As long as value isn't false we will async connect to all genesis pools on startup
-    if (this.ledger.config.connectToIndyLedgersOnStartup) {
+    if (this.ledger.config.connectToLedgersOnStartup) {
```

Comment:
Changing this config will make it a breaking change. I think we can get this into 0.2.3 and we can make the bigger changes with breaking changes (as described in #977) for the 0.3.0 version.